src/mjolnir/landmark_builder.cc

```diff
@@ -153,10 +199,17 @@ std::vector<Landmark> LandmarkDatabase::get_landmarks_in_bounding_box(const doub
   int ret = sqlite3_step(bounding_box_stmt);
   while (ret == SQLITE_ROW) {
     const char* name = reinterpret_cast<const char*>(sqlite3_column_text(bounding_box_stmt, 0));
-    const char* type = reinterpret_cast<const char*>(sqlite3_column_text(bounding_box_stmt, 1));
+
+    LandmarkType type = LandmarkType::null;
+    if (sqlite3_column_type(bounding_box_stmt, 1) != SQLITE_NULL) {
+      type = static_cast<LandmarkType>(sqlite3_column_int(bounding_box_stmt, 1));
+    }
```

Comment:
it shouldn't happen that the type is empty. IIRC we throw away nodes that don't have an `amenity` type no? I'd consider that a bug if that happens, but pretty sure it can't

Reply:
agree, its probably not needed to check this